README.adoc

```diff
@@ -34,14 +34,6 @@ You can take a look at the link:sonar-project.properties[sonar-project.propertie
 - https://docs.sonarcloud.io/advanced-setup/languages/c-c-objective-c/#analysis-steps-using-compilation-database[Using compilation database (compile_commands.json)]
 - https://docs.sonarcloud.io/enriching/test-coverage/test-coverage-parameters/[Test coverage parameters]
 
-= MacOS\XCodeBuild
```

Comment:
Why do you propose to remove this part?

Reply:
After f2f discussion. It is no longer relevant.

Reply:
as agreed, closing this